packages/manager/apps/vrack-services/src/pages/create/ZoneFormField.tsx

```diff
@@ -84,18 +83,18 @@ export const ZoneFormField: React.FC<Props> = ({
                   id={zone.name}
                   key={zone.name}
                   value={zone.name}
-                  name={inputName}
-                  checked={selectedZone === zone.name}
-                  {...(isReadOnly ? { disabled: true } : {})}
+                  name={zoneInputName}
+                  checked={selectedZone === zone.name || undefined}
+                  disabled={isReadOnly || undefined}
```

Comment:
```suggestion
                  disabled={!!isReadOnly}
```

Reply:
on ODS components we have to put undefined to disable a boolean property